sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -1361,30 +1361,35 @@ public void Cfg_Coalesce()
 
             VerifyAllInstructions(branchBlock, "b");
             VerifyAllInstructions(bNullBlock, "c");
-            VerifyAllInstructions(afterOp, "a = b ?? c");
+            VerifyAllInstructions(afterOp, "b", "a = b ?? c");
```

Comment:
this actually makes sense, because `b` is the value that gets assigned to `a` in this block 👍